src/components/MainSection.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState, useEffect } from "react";
+
+interface Image {
+  id_image: string;
+  url_image: string;
+}
+
+interface Item {
+  id: number;
+  content: string;
+  images: Image[];
+}
+
+export function MainSection() {
+  const [data, setData] = useState<Item[]>([]);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
```

Comment:
Copy-pasted code from `Header.tsx`. Create a user hook to load JSON data and reuse it in both Header and MainSection.